tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3746,6 +3747,19 @@ def runTest(self):
 
         send_packet(self, src_port_id, pkt, pkts_num_leak_out)
 
+        if 'hwsku' in self.test_params and self.test_params['hwsku'] in ('Arista-7060X6-64PE-256x200G'):
+            pkt = construct_ip_pkt(default_packet_length,
```

Comment:
this packet length is still 1500, it will be better to change it to a small packet, so we can test against the exact cell number.

Reply:
I'll address all 3 of your comments here:

This latest commit actually reverts it from a previous version where we did have the exact 238 * 64 byte packets. This seems to throw off the rr algorithm and requires increasing `q3_num_of_packets` to account for the fact that an increased number of packets are passing through that queue, which is itself a magic calculation.

My attempt with this commit was to increase the packet size to the default of 1500 and then reduce the number of padding packets by the same factor but it seems like the math doens't quite work out, where you rightfully point out that `238*64/1500 == 10.15`, in testing we know this is incorrect but we're not sure why.